### DISCUSSION TITLE
Hint text

### DISCUSSION THREAD

**Discussion by martinwake** [dwp/design-system#443] [General]
Hello, we'd like to use this open discussion to talk about the DWP Design System Hint text component and how it's being used. Here's some background:

- [Roundup of existing guidance on full stops / no full stops from various public sector design systems](https://github.com/dwp/design-system-community-backlog/discussions/81)
- [Community backlog ticket for Hint text](https://github.com/dwp/design-system-community-backlog/issues/55)

Please share your questions and experiences using the Hint text component here.

**Comment by martinwake** [comment]
From @cristina-agramunt https://github.com/dwp/design-system/issues/240#issuecomment-1023682565

> At the Apply for a Nino we obtained feedback on an accessibility user test that the hint text was not visible. The user had no idea that the hint text existed. It would be worth reviewing the colour of the hint text.

**Comment by martinwake** [comment]
Thanks @cristina-agramunt: can you confirm which colour or colour reference the service uses for hint text?

**Reply by cristina-agramunt** [reply]
Hi Martin, sorry never replied. The tag we were using was -  "govuk-hint" - inspecting in the browser it appears to be: #626a6e

**Comment by martinwake** [comment]
Does a full stop read more awkwardly on "sentence fragments" vs full sentences? For example:

> **Name**
> Your full name.

vs

> **Name**
> Enter your full name.

If this turns out to be a problem, perhaps hint text should always be a full sentence? This would follow [gov.uk service manual guidance](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#capitalisation-and-punctuation) that all text other than headings and input field labels should be a punctuated full sentence.
